pkg/client/image.go

```diff
@@ -229,3 +229,13 @@ func (c *DefaultClient) ImageList(ctx context.Context) ([]apiv1.Image, error) {
 
 	return result.Items, nil
 }
+
+// GetImageReference finds an image if exists and returns if it was found by tag
```

Comment:
nit:

```suggestion
// GetImageRef finds an image if exists and returns if it was found by tag
```

Reply:
Renamed that function actually because there is `GetImageReference` already in that package and it doesn't really reflect what it does.